package.json

```
@@ -1,12 +1,14 @@
 {
   "license": "GPL-3.0",
   "repository": "github:AprilSylph/Palettes-for-Tumblr",
+  "type": "module",
   "scripts": {
     "test": "eslint . && web-ext lint",
     "lint-webextension": "web-ext lint",
     "build-webextension": "web-ext build",
     "sign-webextension": "web-ext sign",
-    "webstore-upload": "chrome-webstore-upload upload --source=src/ --auto-publish"
+    "webstore-upload": "chrome-webstore-upload upload --source=src/ --auto-publish",
+    "updateSystemData": "node dev/update-palette-system-data.js"
```

1. This script does not work on my machine (Ubuntu), I get the following error:
    ```
    Error: Failed to launch the browser process!
    
    (chrome:20156): GLib-GIO-ERROR **: 15:37:05.593: Settings schema 'org.gnome.settings-daemon.plugins.xsettings' does not contain a key named 'antialiasing'
    [0109/153705.599329:ERROR:process_memory_range.cc(75)] read out of range
    [0109/153705.601299:ERROR:process_memory_range.cc(75)] read out of range
    [0109/153705.601308:ERROR:process_memory_range.cc(75)] read out of range
    ```
2. Other scripts are named in snake-case, can we keep that consistent please?
```suggestion
    "update-palette-system-data": "node dev/update-palette-system-data.js"
```